Core/src/IfcGeometryConverter/SolidModelConverter.h

```diff
@@ -2087,7 +2024,7 @@ namespace OpenInfraPlatform
 				if (otherOperand)
 				{
 					carve::geom::aabb<3> aabb;
-					otherOperand->createMeshSetsFromClosedPolyhedrons();
+					otherOperand->createMeshSetsFromclosed_polyhedrons();

```

Comment:
This is a weird function name ...

Reply:
I changed it by mistake. Resolved: 4c40ba7